shielddash/urls.py

```diff
@@ -17,10 +17,11 @@
 from django.contrib import admin
 from django.views.generic import TemplateView
 
-from .studies.views import StudyListView
+from .studies.views import StudyListView, study_detail
 
 
 urlpatterns = [
+    url(r'^studies/(?P<study_id>\d+)/', study_detail, name='study-detail'),
     url(r'^studies/', StudyListView.as_view(), name='study-list'),
```

Comment:
Pretty sure the `StudyListView.as_view` supplies its own `study-detail` so I'm "overriding" that but putting mine first. If there's a better way to do this with DRF let me know.